examples/simple-platform/run-example.md

```diff
@@ -0,0 +1,144 @@
+# Running the Platform Example Locally
+
+## Setup
+
+### Create a GitHub development application for the `account-app` OAuth process
+
+At this time, the accounts app supports account creation and authentication 
+via GitHub only. In order to run a user authentication loop, you need to 
+create a new GitHub OAuth app and configure it to connect to the 
+`web-auth-endpoint` node.
+
+1. Log into [github.com](https://github.com) and navigate to Settings > 
+Developer settings. 
+1. Under OAuth Apps, create a new OAuth App and give it any application name, 
+e.g. Interbit Test App.
+1. The Authorization callback URL is `http://localhost:8888/oauth/github`.
```

Comment:
Some points that should be conveyed here:
The callback URL is case sensitive.
`localhost` is only for development. 
The port number is that specified for running the `web-auth-endpoint` node locally.